media/src/io/sample_builder/sample_builder_test.rs

```diff
@@ -813,7 +813,7 @@ pub fn test_sample_builder() {
                 data: bytes!(4, 5),
                 duration: Duration::from_secs(2),
                 packet_timestamp: 4000,
-                prev_dropped_packets: 13,
```

Comment:
This was actually wrong. The total number of dropped packets is `12`(5000 through 5012).

Reply:
Actually it was correct since it's 5000 _through_ 5012, not 5000 _to_ 5012.

I'll fix